tests/queries/0_stateless/02335_column_ttl_expired_column_optimization.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+$CLICKHOUSE_LOCAL --path "$CLICKHOUSE_TEST_UNIQUE_NAME" -nm -q "
```

Comment:
This test creates a file and never deletes it. Also probably it should be created in `$CLICKHOUSE_TMP` folder, not in the current directory.

Reply:
@azat ^